homepage.js

```diff
@@ -24,11 +24,13 @@
      * @param events: JSON event object fetched from Bedeworks
      */
     var initializeEventsHomepage = function(eventsJson) {
-        var allEvents = CTLEventsManager.loadEvents(eventsJson);
-        jQuery('#homepage-calendar-listing').append(renderEvents(allEvents, ITEMS_TO_DISPLAY));
+        var today = new Date();
+        var allHomepageEvents = CTLEventsManager.loadEvents(eventsJson, today);
+        jQuery('#homepage-calendar-listing').append(renderEvents(allHomepageEvents, ITEMS_TO_DISPLAY));
     };
 
     $(document).ready(function() {
+        $('head').append('<style type="text/css"> .event{margin-top: 2em;} .event:first-child {margin-top: 0em;}');
```

Comment:
hmmm, are we not able to edit the CSS on this page? it would be nice if we didn't have to put this styling in JS code.